subprojects/docs/src/docs/userguide/commandLineInterface.adoc

```diff
@@ -0,0 +1,520 @@
+// Copyright 2017 the original author or authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+[[command_line_interface]]
+== Command-Line Interface
```

Comment:
:o: There are a couple of section in here that makes me think that "Command-Line Interface" is not the right name for this chapter.

Reply:
I have removed "Running Tests", "System Properties", and "Environment Variables" and will populate different other chapters with their contents. Better now?

Reply:
Looks like "Running tests" is still in there. IMHO that's still a JVM feature right now and doesn't apply to other contexts e.g. native AFAIK. But generally better now. I think we can keep the title.

Reply:
Right, I re-added "Running tests". I had thought that the new native stuff used `test` but I'm not sure.